collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -213,6 +215,15 @@ protected FECWrapper doInBackground(String... path) {
         return data;
     }
 
+    private String getFormFileName(File formXml) {
```

Comment:
Is the method's name right, though? This is not returning the filename, but the filename without extension. I think something like filenameBase would work better.